### PR TITLE
Edited DB pull inside ticketController.php, Edited innerHTML for repo…

### PR DESCRIPTION
**Pull request by LewisWelch94**
…rter&assigne to now show username wrapped in brackets on peopleController.

### CODE REVIEW

**Comment by LewisWelch94** [comment]
This is Lmaoo170_1 & Lmaoo170_2, not Lmaoo170. started wrong task and couldnt rename remote branch name.

File: Script/peopleController.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ function loadAssignee()
     if (response.data.length == 0) return;
 
     var res = response.data;
-    $("#assignee").html(res[0].forename + " " + res[0].surname);
+    $("#assignee").html(res[0].forename + " " + res[0].surname + " " + "(" + res[0].username + ")");
```
Comment (OnikNoor97) [inline Script/peopleController.js:34]: Use Template Literals like in the pro video, check the URL for better understanding
https://blog.webdevsimplified.com/2020-03/tagged-template-literals/

File: Script/peopleController.js [hunk 2]
```diff
@@ -43,7 +43,7 @@ function loadReporter()
   .then(response =>
   {
     var res = response.data;
-    $("#reporter").html(res[0].forename + " " + res[0].surname);
+    $("#reporter").html(res[0].forename + " " + res[0].surname + " " + "(" + res[0].username + ")");
```
Comment (OnikNoor97) [inline Script/peopleController.js:46]: Same as above

File: Ticket/ticketController.php [hunk 3]
```diff
@@ -150,7 +150,7 @@ function loadAssignee($ticketId)
 {
     $pdo = logindb('user', 'pass');
     $pdo->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_OBJ);
-    $stmt = $pdo->prepare("SELECT user.forename, user.surname, user.userId FROM ticket 
+    $stmt = $pdo->prepare("SELECT user.forename, user.surname, user.username, user.userId FROM ticket
```
Comment (OnikNoor97) [inline Ticket/ticketController.php:153]: Very good!

**Pull request merged**